src/Geometry.h

```diff
@@ -27,6 +27,19 @@
 class Cmfd;
 
 /**
+ * @enum ringifyType
+ * @brief The method to use when subdividing cells into rings
+ */
+enum ringifyType{
```

Comment:
Yikes, thanks for pointing that out! Perhaps you could add the space in there as well?